src/screens/ErrorPage.js

```diff
@@ -0,0 +1,26 @@
+import { Button} from "../components";
+import { render } from "../shared/dom";
+import { routes } from "../route";
+import { redirect } from "..";
+
+
+
+const template = `
+<div class="notFoundPage__container">
+    <h1>Oops!</h1>
+    <h2>Page not found</h2>
+    <h3>404</h3>
+    <p>the page you are looking for 
```

Comment:
This should start with the capital letter

Reply:
Done